sdk/storage/azdatalake/internal/generated_blob/autorest.md

```diff
@@ -347,11 +347,13 @@ directive:
 
 ``` yaml
 directive:
-  - from: zz_service_client.go
-    where: $
-    transform: >-
-      return $.
-        replace(/req.Raw\(\).URL.RawQuery \= reqQP.Encode\(\)/, `req.Raw().URL.RawQuery = strings.Replace(reqQP.Encode(), "+", "%20", -1)`)
+- from: 
+  - zz_service_client.go
+  - zz_container_client.go
```

Comment:
Same as previous, this change should come in both these files in blob's generated code

Reply:
Since the transform already existed in zz_service_client.go, there is no diff. The change is still there. https://github.com/Azure/azure-sdk-for-go/blob/0de654744232cf8aaed01dee831bb81ea1f97b02/sdk/storage/azdatalake/internal/generated/zz_service_client.go#L52